charts/prometheus-customizations/Chart.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v2
+name: prometheus-customizations
+description: An opinionated helm chart to setup our prometheus instances. Requires externalsecrets, configmapsecrets, sets the prom config to a particular value.
```

Comment:
not to be dense, but this chart *only* generates the prometheus config, it does not actually deploy prometheus,  yeah?

Reply:
Yup, I use 'setup' to mean configures but doesn't deploy, but that's maybe a little overloaded, would something like:

"helm chart to generate a config for our prometheus setup" be better?